db/seeds.rb

```diff
@@ -13,7 +13,8 @@
 
 50.times do
   Tweet.create!(user_id: rand(1...21),
-                text: Faker::Lorem.sentence
+                text: Faker::Lorem.sentence,
+                created_at: (rand*10).days.ago
```

Comment:
Not sure if you realized, but this is actually giving you different times, too:

````ruby
irb(main):001:0> (rand*10).days.ago
=> Sun, 17 Jan 2016 20:02:20 UTC +00:00
irb(main):002:0> (rand*10).days.ago
=> Sat, 16 Jan 2016 00:43:29 UTC +00:00
irb(main):003:0> (rand*10).days.ago
=> Fri, 15 Jan 2016 18:04:31 UTC +00:00
irb(main):004:0> (rand*10).days.ago
=> Sat, 16 Jan 2016 05:45:50 UTC +00:00
````
I think that's fine. If you don't want that, you can do:
````ruby
irb(main):013:0> rand(10).days.ago
=> Sun, 17 Jan 2016 18:39:59 UTC +00:00
irb(main):014:0> rand(10).days.ago
=> Tue, 12 Jan 2016 18:39:59 UTC +00:00
irb(main):015:0> rand(10).days.ago
=> Mon, 11 Jan 2016 18:39:59 UTC +00:00
````